mmdet3d/models/voxel_encoders/pillar_encoder.py

```diff
@@ -104,14 +104,13 @@ def forward(self, features, num_points, coors):
             features_ls.append(f_cluster)
 
         # Find distance of x, y, and z from pillar center
+        dtype = features.dtype
         if self._with_voxel_center:
-            f_center = features[:, :, :2]
-            f_center[:, :, 0] = f_center[:, :, 0] - (
-                coors[:, 3].type_as(features).unsqueeze(1) * self.vx +
-                self.x_offset)
-            f_center[:, :, 1] = f_center[:, :, 1] - (
-                coors[:, 2].type_as(features).unsqueeze(1) * self.vy +
-                self.y_offset)
+            f_center = torch.zeros_like(features[:, :, :2])
```

Comment:
Add an option to decide whether to use the new behavior or the original behavior